### ISSUE TITLE
Romanian language

### ISSUE THREAD

**Issue by linman1978** [RaduBolbo/F5-TTS-Emotional-CFG#2]
How can be finetuned o romanian language?
I created an audio dataset of approximately 20 hours with audio from romanian librivox. (public domain)

**Comment by RaduBolbo** [comment]
Hello!

Our implementation only supports emotional control in English, but the training code could potentially be adapted for any other language. However, we fine-tuned on a small dataset and relied on the fact that the model had already been trained for English. For training in another language probably the original non-emotional training in [F5-TTS](https://github.com/SWivid/F5-TTS?tab=readme-ov-file#f5-tts-a-fairytaler-that-fakes-fluent-and-faithful-speech-with-flow-matching) is the way to go.

Are you interested in training with emotion (the Romanian dataset has emotion labels) or keeping the emotion control? If you are going to also train on Romanian without emotion labels, part of the emotion control will likely be lost, as our method is more of a fine-tune on a reduced emotion labeled dataset after the model has already good modeling of that language. 

There are several implementations of the original [F5-TTS](https://github.com/SWivid/F5-TTS?tab=readme-ov-file#f5-tts-a-fairytaler-that-fakes-fluent-and-faithful-speech-with-flow-matching) that are trained for other languages such as [Spanish-F5](https://github.com/jpgallegoar/Spanish-F5).
